Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.24.5 as builder
```

Comment:
This should be done in a separate PR.

Reply:
Moved to https://github.com/qbee-io/qbee-agent/pull/108